app/src/Login/Login.css

```diff
@@ -0,0 +1,45 @@
+.loginpage {
+  background-color: #f0f0f0;
+  position: absolute;
+  top: 15%;
+  left: 30%;
+  right: 30%;
+  height: 80vh;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.loginpage h1, .signuppage h1 {
+  margin-top: 10%;
+  margin-bottom: 6%;
+  user-select: none;
+}
+
+.loginpage p {
+  font-size: 2.3vh;
+  user-select: none;
+}
+
+#forgot-psw, #sign-up-button, #sign-in-button {
+  color: #2400ff;
+  font-size: 2vh; 
+  font-weight: 700;
+  margin: 2%;
+  border: none;
+  cursor: pointer;
+}
+
+.signuppage {
+  background-color: #f0f0f0;
+  position: absolute;
+  top: 15%;
+  left: 30%;
+  right: 30%;
+  height: 80vh;
```

Comment:
Same thing here: min-height and min-width: fit-content to prevent this
<img width="324" alt="Screenshot 2023-11-06 at 10 19 01 PM" src="https://github.com/twang7310/rate-my-cse/assets/131730445/2351b6ea-4066-4347-8f51-59794942fb68">